src/image/release.py

```diff
@@ -75,7 +76,7 @@
         all_releases[track] = {}
 
     for risk, value in risks.dict(exclude_none=True).items():
-        if risk == "end-of-life":
+        if risk in ["end-of-life", "end_of_life"]:
```

Comment:
Always at the moment. As a result of that, "end_of_life" is considered as an "unknown risk" and skipped altogether. The mechanism of adding eol values to the releases dict is already there before this PR, but this value has been always skipped because the field comes as "end_of_life".